src/pages/gen2/start/account-setup/index.mdx

```diff
@@ -9,105 +9,206 @@ export function getStaticProps(context) {
 
-Configure a profile by running the following command:
+<Callout info>
 
-```bash
-npx amplify configure profile --name <value>
-```
+The information presented below can also be found on the [official AWS documentation for IAM Identity Center](https://docs.aws.amazon.com/singlesignon/latest/userguide/what-is.html).
 
-`npx amplify configure profile` will ask you if you already have setup IAM user credentials. Enter `Yes`.
+</Callout>
 
-Next, enter the credentials of the existing IAM user (this is their access key and secret access key), and then select the AWS region to use with the profile.
+1. Sign in to the [AWS Management Console](https://console.aws.amazon.com/)
+2. Navigate to the [IAM Identity Center page](https://console.aws.amazon.com/singlesignon/home)
 
-![](/images/gen2/account-setup/profile2.png)
+  ![AWS Console with search opened for "IAM Identity Center"](/images/gen2/account-setup/console-search-iam-identity-center.png)
 
-<Accordion title='Single Sign On (SSO) support' eyebrow='Learn more'>
-  AWS Amplify (Gen 2) enhances your user experience with Single Sign-On (SSO) authentication powered by AWS IAM Identity Center. AWS Amplify (Gen 2) now adheres to and works out of the box with the default [credential provider chain](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/setting-credentials-node.html#credchain).
+3. Select **Enable**
 
-You can get started with implementing SSO with the AWS CLI for your teams or organizations by referring to the [AWS IAM Identity Center documentation](https://docs.aws.amazon.com/sdkref/latest/guide/feature-sso-credentials.html).
+  ![AWS IAM Identity Center page indicating the "enable" button](/images/gen2/account-setup/sso-enable.png)
 
-</Accordion>
+4. A dialog will open and prompt for "how you would like to configure IAM Identity Center in your AWS environment", choose "AWS Organizations" and select **Continue**
+
+  ![AWS IAM Identity Center enable dialog with AWS Organizations, indicating to "continue"](/images/gen2/account-setup/sso-enable-dialog.png)
+
+5. After the AWS Organization is created and IAM Identity Center is enabled you are presented with a dashboard. In the navigation pane, select **Permission sets**
+
+  ![AWS IAM Identity Center dashboard indicating "permission sets" in the navigation pane](/images/gen2/account-setup/sso-dashboard-highlight-permission-sets.png)
+
+6. Select **Create permission set**
+7. When prompted for the permission set type, choose **Predefined permission set** and the **Administrator Access** policy. Then select **Next**
```

Comment:
Should this be the `AmplifyBackendDeployFullAccess` policy instead?

Reply:
permission sets are a bit different, but we can probably show scoping sessions down with a session role ![image](https://github.com/aws-amplify/docs/assets/5033303/3c341fbe-2b3e-4b1d-9199-378b020c5fe3)

Reply:
Ah gotcha. Yeah I guess admin access is the best option in this case.